ballast_info.py

```diff
@@ -67,10 +69,42 @@ def ballast_info(glider_datasets, threshold=420):
     
     for name, ds in ds_dict.items():
     
-        #Max and min ballast values per mission and total dives
+        #Max and min ballast values per mission and total dives + total volume pumped for mission
         max_ballast.append(np.nanmax(ds.ballast_pos))
         min_ballast.append(np.nanmin(ds.ballast_pos))
         total_dives.append(ds['dive_num'].values.max())
+
+        max_depth.append(int(np.nanmax(np.abs(ds.depth))))
+        
+        if np.diff(ds.time).mean()<10**8:
+            test=ds.thin({"time": 70}) #.isel(obs=ds.dive_num==528)
+        else:
+            test=ds.thin({"time": 15}) #.isel(obs=ds.dive_num==528)
+        ballast = test.isel(time=np.isfinite(test.ballast_pos.values))
+        ballast= ballast.isel(time=np.isfinite(ballast.ballast_cmd.values))
+        
+        pos=ballast.ballast_pos.values
+        cmd = ballast.ballast_cmd.values
+        
+        #ballast = ballast[~np.isnan(ballast)]
+        pos_pre = pos.copy()[:-1]
+        pos_post = pos.copy()[1:]
+        pos_diff = pos_post - pos_pre
+        
+        #cmd = cmd[~np.isnan(ballast)]
+        cmd_pre = cmd.copy()[:-1]
+        cmd_post = cmd.copy()[1:]
+        cmd_diff = cmd_post - cmd_pre
+
+        noise_threshold_one=5
```

Comment:
it would be nice to make this pump noise threshold available as as kwarg, like we do with threshold=420